src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  fahrenheit: 60,
+  city: 'Edmonds',
+  sky: 'Sunny',
+};
+let num = state.fahrenheit;
+const tempUp = (event) => {
+  state.fahrenheit += 1;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const increase = document.getElementById('f');
+  increase.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return increase.textContent;
+};
+
+const tempDown = (event) => {
+  state.fahrenheit--;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const decrease = document.getElementById('f');
+  decrease.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return decrease.textContent;
+};
+
+const cityInput = (event) => {
+  let cityInput = document.getElementById('input').value;
+  state.city = cityInput;
+  let cityHeader = document.getElementById('city');
+  cityHeader.textContent = `${state.city}`;
+  getCityWeather();
+};
+
+const tempChangeColor = (num) => {
+  let element = document.getElementById('temperatureContainer');
+  if (state.fahrenheit >= 80) {
+    element.style.backgroundColor = 'red';
+  } else if (80 > state.fahrenheit && state.fahrenheit >= 70) {
+    element.style.backgroundColor = 'orange';
+  } else if (70 > state.fahrenheit && state.fahrenheit >= 60) {
+    element.style.backgroundColor = 'yellow';
+  } else if (60 > state.fahrenheit && state.fahrenheit >= 50) {
+    element.style.backgroundColor = 'green';
+  } else if (50 > state.fahrenheit && state.fahrenheit >= 33) {
+    element.style.backgroundColor = 'teal';
+  } else if (state.fahrenheit <= 32) {
+    element.style.backgroundColor = 'blue';
+  }
```

Comment:
Check out the logic in tempChangeColor(), see how similar the branches are of the conditional statement. We look for whether the temperature is within a range, and pick a color accordingly. What if we had a list of records that we could iterate through to find the values. We could set up something like

```javascript
const TEMP_COLORS = [ 
  { upperBound: 49, color: "teal"}, 
  { upperBound: 59, color: "green"}, 
  { upperBound: 69, color: "yellow"}, 
  { upperBound: 79, color: "orange"}, 
  { upperBound: 80, color: "red"}
];
```

Then your method would iterate through `TEMP_COLORS` and find the first record that has an upper bound higher than our temperature, then use it as the source of picking the color which would help make the function a bit more concise.

This could accommodate a scenario where you might have even more colors in the future, which would prevent your conditional statement from being a really long block of if/elif/elif and so on